src/public/parts/shells/config/public/index.html

```
@@ -35,7 +35,7 @@ <h1 class="text-3xl font-bold mb-4" data-i18n="part_config.pageTitle"></h1>
 					<div class="breadcrumbs overflow-x-auto">
 						<ul id="partpath-breadcrumb" class="flex-nowrap"></ul>
 					</div>
-					<img src="https://api.iconify.design/line-md/menu.svg" class="w-5 h-5 ml-auto" />
+					<img src="https://api.iconify.design/line-md/menu.svg" class="w-5 h-5 ml-auto" aria-hidden="true" />
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)